src/Commands/InstallCommand.php

```diff
@@ -134,4 +165,23 @@ public function handle(Filesystem $filesystem)
 
         $this->info('Successfully installed Voyager! Enjoy');
     }
+
+    private function addNamespaceIfNeeded($seeds, Filesystem $filesystem)
+    {
+        if ($this->seedFolder != 'seeders') {
+            return;
+        }
+
+        $seeds->each(function ($file) use ($filesystem) {
+            $path = database_path('seeders').'/'.$file->getFilename();
+
+            $stub = str_replace(
+                "<?php\n\nuse",
```

Comment:
But we are using this on our own seeders only.

Reply:
Yup, but it's something easily missed in PR reviews.  Anyone could add a new seeder as part of a feature change/addition, and we currently have no controls around code style for them (as far as I know)

Reply:
I'm not against making it better but I can't think of a quick way because you'll still need to understand where to insert the namespace.
In the other hand I'm not really sure we need to change it, if for any reason adding the namespace is not going to work when adding some seeder whoever is coding the seeder should already notice that.